docs/source/user-guide/dataframe.md

```diff
@@ -0,0 +1,278 @@
+
+## Math Functions
+
+In addition to the math functions listed here, some Rust operators are implemented for expressions, allowing
+expressions such as `col("a") + col("b")` to be used.
+
+| Function              | Notes                                             |
+| --------------------- | ------------------------------------------------- |
+| abs(x)                | absolute value                                    |
+| acos(x)               | inverse cosine                                    |
+| asin(x)               | inverse sine                                      |
+| atan(x)               | inverse tangent                                   |
+| atan2(y, x)           | inverse tangent of y / x                          |
+| ceil(x)               | nearest integer greater than or equal to argument |
+| cos(x)                | cosine                                            |
+| exp(x)                | exponential                                       |
+| floor(x)              | nearest integer less than or equal to argument    |
+| ln(x)                 | natural logarithm                                 |
+| log10(x)              | base 10 logarithm                                 |
+| log2(x)               | base 2 logarithm                                  |
+| power(base, exponent) | base raised to the power of exponent              |
+| round(x)              | round to nearest integer                          |
+| signum(x)             | sign of the argument (-1, 0, +1)                  |
+| sin(x)                | sine                                              |
+| sqrt(x)               | square root                                       |
+| tan(x)                | tangent                                           |
+| trunc(x)              | truncate toward zero                              |
+
+## Conditional Expressions
+
+| Function | Notes                                                                                                                                                                                                    |
+| -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| coalesce | Returns the first of its arguments that is not null. Null is returned only if all arguments are null. It is often used to substitute a default value for null values when data is retrieved for display. |
+| case     | CASE expression. Example: `case(expr).when(expr, expr).when(expr, expr).otherwise(expr).end()`.                                                                                                          |
+| nullif   | Returns a null value if value1 equals value2; otherwise it returns value1. This can be used to perform the inverse operation of the `coalesce` expression.                                               |
```

Comment:
```suggestion
| nullif   | Returns a null value if `value1` equals `value2`; otherwise it returns `value1`. This can be used to perform the inverse operation of the `coalesce` expression.                                               |
```